src/proxy_lib/proxy_lib.c

```diff
@@ -145,9 +143,8 @@ void proxy_lib_create_common(void) {
         exit(-1);
     }
 
-    umf_result =
-        umfPoolCreate(umfPoolManagerOps(), OS_memory_provider, NULL,
-                      UMF_POOL_CREATE_FLAG_DISABLE_TRACKING, &Proxy_pool);
+    umf_result = umfPoolCreate(umfPoolManagerOps(), OS_memory_provider, NULL, 0,
+                               &Proxy_pool);
```

Comment:
Can you use the same coding style for variable names?
`Proxy_poo` -> `proxy_pool`
`OS_memory_provider` -> `os_memory_provider`

Reply:
these are globals, so they start with a capital letter - see how they are defined and used in the rest of the file